samples/bot-receive-channel-messages-withRSC/csharp/README.md

```diff
@@ -0,0 +1,97 @@
+﻿﻿﻿Using this C# sample, a bot can receive all channel messages with RSC without @mention.
+For reference please check [Receive Channel messages with RSC](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/channel-messages-with-rsc)
+
+This feature shown in this sample is currently available in Public Developer Preview only.
+
+## Key features
+
+- Showing messages based on option selected
+
+![Channel messages](ReceiveMessagesWithRSC/Images/botWithRSCFlow.png)
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 3.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+- Publicly addressable https url or tunnel such as [ngrok](https://ngrok.com/) or [Tunnel Relay](https://github.com/OfficeDev/microsoft-teams-tunnelrelay) 
+
+## Setup
+
+1. Run ngrok - point to port 3978
+
+```bash
+# ngrok http -host-header=rewrite 3978
+```
+
+2. Create a Bot Registration
+   In Azure portal, create a [Bot Framework registration resource](https://docs.microsoft.com/en-us/azure/bot-service/bot-builder-authentication?view=azure-bot-service-4.0&tabs=csharp%2Caadv2#create-the-resource).
+
+3. Modify the `manifest.json` in the `/AppManifest` folder and replace the `{{BOT-ID}}` with the id from step 2.
+
+4. Zip the contents of `AppManifest` folder into a `manifest.zip`, and use the `manifest.zip` to deploy in app store or add to Teams as in step 7.
+
+5. Modify the `/appsettings.json` and fill in the `{{ Bot Id }}`,`{{ Bot Password }}` with the id from step 2.
+
+- __*This step is specific to Teams.*__
+    - **Edit** the `manifest.json` contained in the  `AppManifest` folder to replace your Microsoft App Id (that was created when you registered your bot earlier) *everywhere* you see the place holder string `<<YOUR-MICROSOFT-APP-ID>>` (depending on the scenario the Microsoft App Id may occur multiple times in the `manifest.json`) also update the `<<DOMAIN-NAME>>` with the ngrok URL`
+    
+    - **Zip** up the contents of the `appPackage` folder to create a `manifest.zip`
+    - **Sideload** in a team to test
+         - Select or create a team
+         - Select the ellipses **...** from the left pane. The drop-down menu appears.
+         - Select **Manage Team**, then select **Apps** 
+         - Then select **Upload a custom app** from the lower right corner.
+         - Then select the `manifest.zip` file from `appPackage`, and then select **Add** to add the bot to your selected team.
+    
+## To try this sample
```

Comment:
Could you please include steps to clone the repo? Also I think `cd # ReceiveMessagesWithRSC` should be `cd  ReceiveMessagesWithRSC`.

Reply:
Added in try this sample section